Toolset/palettes/start center/revStartCenterBehavior.livecodescript

```diff
@@ -104,11 +104,13 @@ on ideOpenStack pTarget
       if tLine ends with ".livecode" or tLine ends with ".livecodescript" or \
             tLine ends with ".rev" then 
          put the substacks of stack tLine into tSubstacks
+         // Unload Plugin from memory
+         delete stack tLine
```

Comment:
@livecodepanos Will this change IDE behavior if you always delete the plugin stack, even if it was already in memory?

Reply:
@trevordevore hmm good point. If the stack is already in memory but not open, then I believe this change will not have any other unwanted consequences. 

But if the stack is already open, then this line will cause the stack to close, which is something we don't want to. I guess we should do something like:

```
if not (the short name of stack tLine is in the openstacks) then
   delete stack tLine
end if
```

Reply:
@livecodepanos This is better. Ideally it wouldn't alter the state of the stack at all (if the stack is among the mainstacks it should remain among the mainstacks) but that isn't possible with existing syntax. Purging a stack from memory can affect script local variables. `the stacks` returns stack filenames that have open stacks but there is no equivalent for filenames of `mainstacks`.